src/CMakeLists.txt

```diff
@@ -1038,6 +1038,12 @@ if(WITH_RADOSGW)
   add_dependencies(vstart radosgw radosgw-admin)
 endif(WITH_RADOSGW)
 
+if(WITH_LTTNG)
+  add_dependencies(vstart rados_tp)
+  add_dependencies(vstart os_tp)
+  add_dependencies(vstart osd_tp)
+  add_dependencies(vstart rbd_tp)
+endif(WITH_LTTNG)
```

Comment:
does it work if you put these in `tracing/CMakeLists.txt` instead? that would make it easier to keep these dependencies up-to-date if we add more tracing libraries in the future